.github/scripts/build.sh

```diff
@@ -29,7 +29,7 @@ if [[ "$PLATFORM" =~ cygwin ]]; then
     .github/scripts/toolchain/patch-cygwin.sh 1
 fi
 
-if [ "$RUN_BOOTSTRAP" = 1 ]; then
+if [[ "$RUN_BOOTSTRAP" = 1 || "$RESET_SOURCES" = 1 ]]; then
```

Comment:
Why this check is needed?

Reply:
When the sources are reset before applying the paches, the symlinks to the `gmp`, `mprf`, etc. need to be restored.